openmdao/recorders/tests/test_sqlite_reader.py

```diff
@@ -912,6 +912,83 @@ def test_load_solver_cases(self):
 
         _assert_model_matches_case(case, model)
 
+
+    def test_recording_option_precedence_driver_cases(self):
+        prob = Problem()
```

Comment:
Feature tests need all relevant imports in the body of the test. This should fail the doctest without them.